openpype/hosts/blender/blender_addon/addons/DeadlineBlenderScenePrepare.py

```diff
@@ -117,6 +121,17 @@ def _set_common_infos(item, render_property):
             _set_common_infos(item, render_property)
 
 
+def collect_render_layers():
+        for render_layer_name in get_render_layers_names():
+            item = bpy.context.window_manager.render_layers_to_use.add()
+            item.name = render_layer_name
+            item.value = bpy.context.scene.view_layers[render_layer_name].use
+
+
+def render_layers_needs_update():
```

Comment:
Why just checking the len not the layers by creating python sets?

Reply:
It would probably be safer to compare sets, you're right, i'll update it